src/main/java/it/gov/pagopa/pu/bff/connector/process_executions/client/ExportFileSearchClient.java

```diff
@@ -0,0 +1,42 @@
+package it.gov.pagopa.pu.bff.connector.process_executions.client;
+
+import it.gov.pagopa.pu.bff.connector.process_executions.config.ProcessExecutionsApisHolder;
+import it.gov.pagopa.pu.bff.dto.ExportFileFiltersDTO;
+import it.gov.pagopa.pu.bff.util.PageUtils;
+import it.gov.pagopa.pu.processexecutions.dto.generated.PagedModelExportFile;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class ExportFileSearchClient {
+
+    private final ProcessExecutionsApisHolder processExecutionsApisHolder;
+
+    public ExportFileSearchClient(
+      ProcessExecutionsApisHolder processExecutionsApisHolder) {
+        this.processExecutionsApisHolder = processExecutionsApisHolder;
+    }
+
+  public PagedModelExportFile getExportFiles(ExportFileFiltersDTO exportFileFilters, String operatorExternalId, Pageable pageable, String accessToken) {
+    try {
+      return processExecutionsApisHolder.getExportFileSearchControllerApi(accessToken)
+        .crudExportFilesFindByOrganizationIDFlowTypeCreateDate(
+          String.valueOf(exportFileFilters.getOrganizationId()),
+          exportFileFilters.getFlowFileType().toString(),
+          exportFileFilters.getCreationDate().getFrom(),
+          exportFileFilters.getCreationDate().getTo(),
+          exportFileFilters.getStatus().name(),
+          exportFileFilters.getFileName(),
+          operatorExternalId,
+          PageUtils.getPageNumber(pageable),
+          PageUtils.getPageSize(pageable),
+          PageUtils.getSortList(pageable));
+    } catch (Exception e) {
+      log.error("Unexpected error while retrieving ingestion flow files", e);
+      throw e;
+    }
```

Comment:
As reported yesterday during SU and documented on Confluence, this is not more necessary, please remove this try/catch

https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1540096383/Logging#2.3.-Log-body-di-risposta-delle-chiamate-REST-in-errore